tests/orgsGetByTag.test.js

```diff
@@ -64,3 +144,52 @@ test('Get all the active organisations, associated with a specific tag', functio
     });
   });
 });
+
+test('When no orgs are found to match a specific tag', function (t) {
+  var tagId = 2;
+  init(config, function (err, server, pool) {
+    if (err) {
+      console.log('error initialise server', err);
+      return t.fail();
+    }
+
+    server.inject({
+      method: 'GET',
+      url: '/orgsGetByTag?tags=' + tagId
+    }, function (res) {
+      var expected = {
+        filter_tag: 'Corporate',
+        orgs: [{
+          id: null,
+          name: null,
+          logo_url: null,
+          active: null
+        }]
```

Comment:
I've actually changed this so it's now `orgs: null` if no orgs are returned. Am I right in guessing you'd still prefer an empty array? I guess it makes sense given the nature of the pg responses in general.